openhtf/__init__.py

```diff
@@ -86,7 +86,7 @@ def __call__(self, test_record):  # pylint: disable=invalid-name
 
 class TestPhaseOptions(mutablerecords.Record(
     'TestPhaseOptions', [], {'timeout_s': None, 'run_if': None})):
-  """
+  """Test Phase Options object.
```

Comment:
It was to suppress a lint error. If you have a better docstring, I'm all ~~ears~~ eyes